pkg/rook/static/flexmigrator/kustomize/overlays/kurl/flex-migrator.patch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-migrator
+  namespace: rook-ceph
+spec:
```

Comment:
This may need nodeAffinity since the host path binary is only available on the node we are running the script.